data-pipeline/tests/test_trace_exporter.rs

```diff
@@ -222,6 +223,8 @@ mod tracing_integration_tests {
             let trace_exporter = builder.build().expect("Unable to build TraceExporter");
 
             let data = get_v05_trace_snapshot_test_payload();
+            let data = data.as_ref();
+
             let response = trace_exporter.send(data, 1);
```

Comment:
style nit. If you agree, you can apply this to the other integration tests too. 

```suggestion
            let response = trace_exporter.send(data.as_ref(), 1);
```